source/class/qx/ui/form/FileSelectorButton.js

```diff
@@ -60,9 +60,19 @@
  * [MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/File_API/Using_files_from_web_applications)
  */
 
-let InputElementIdCounter = 0;
 qx.Class.define("qx.ui.form.FileSelectorButton", {
   extend: qx.ui.form.Button,
+
+  statics: {
+    _FileInputElementIdCounter: 0,
+    _FileInputElementIdPrefix: "qxFileSelector_",
```

Comment:
Good catch, that it was referencing a global variable. This is a good change.

By convention, qooxdoo uses an uppercase first letter to indicate that the variable contains a class reference, e.g., one might do, `Object : qx.core.Object`... but variables not containing a class reference have a lower-case first letter. Would you please change these to `_fileInput...` instead of `_FileInput...`